psalm.xml

```diff
@@ -59,6 +59,11 @@
                 <file name="src/Resources/config/services.php" />
             </errorLevel>
         </MixedReturnStatement>
+        <PossiblyInvalidArgument errorLevel="error">
+            <errorLevel type="suppress">
+                <referencedFunction name="Symfony\Component\Security\Http\Authenticator\Passport\SelfValidatingPassport::__construct" />
+            </errorLevel>
+        </PossiblyInvalidArgument>
     </issueHandlers>
```

Comment:
hmm, these shouldn't be declared here, we should create a baseline instead and update it if it's not possible to fix the issue.

create a baseline: `php vendor/bin/psalm --set-baseline=.psalm.baseline.xml`
update the baseline: `php vendor/bin/psalm --update-baseline`